docs/references/ios/sign-in-with-apple.mdx

```diff
@@ -26,31 +25,14 @@ This guide will teach you how to add native Sign in with Apple to your Clerk app
   To obtain an Apple ID Credential, you can do one of the following:
 
   - Use one of [Apple's built-in Sign in with Apple buttons](https://developer.apple.com/documentation/sign_in_with_apple/displaying_sign_in_with_apple_buttons_in_your_app).
-  - Obtain it manually by following [the Apple docs](https://developer.apple.com/documentation/sign_in_with_apple/
+  - Obtain it manually by following [the Apple docs](https://developer.apple.com/documentation/sign_in_with_apple/)
 
   > [!NOTE]
   > You must set the nonce property of the `ASAuthorizationAppleIDRequest` when requesting an Apple ID Credential in order to authenticate with Clerk.
 
   ### Sign In / Sign Up
 
-  Once you have obtained your [Apple ID Credential](https://developer.apple.com/documentation/authenticationservices/asauthorizationappleidcredential), you can use it to authenticate with Clerk by calling `SignIn.signInWithAppleIdToken()` or `SignUp.signUpWithAppleIdToken()`
-
-  ```swift {{ filename: 'SignIn' }}
-  static func signInWithAppleIdToken(
-    idToken: String,
-    code: String? = nil
-  ) async throws -> ExternalAuthResult?
-  ```
-
-  ```swift {{ filename: 'SignUp' }}
-  static func signUpWithAppleIdToken(
-    idToken: String,
-    code: String? = nil,
-    firstName: String? = nil,
-    lastName: String? = nil,
-    captchaToken: String? = nil
-  ) async throws -> ExternalAuthResult?
-  ```
+  Once you have obtained your [Apple ID Credential](https://developer.apple.com/documentation/authenticationservices/asauthorizationappleidcredential), you can use it to authenticate with Clerk by calling [`SignIn.signInWithAppleIdToken()`](https://swiftpackageindex.com/clerk/clerk-ios/main/documentation/clerksdk/signin/signinwithappleidtoken\(idtoken:code:\)) or [`SignUp.signUpWithAppleIdToken`](https://swiftpackageindex.com/clerk/clerk-ios/main/documentation/clerksdk/signup/signupwithappleidtoken\(idtoken:code:firstname:lastname:captchatoken:\))
```

Comment:
these links don't work
<img width="1512" alt="Screenshot 2024-08-06 at 16 25 10" src="https://github.com/user-attachments/assets/cabd8aef-ab6e-4b52-8a15-dd44adf50c84">

<img width="1512" alt="Screenshot 2024-08-06 at 16 25 13" src="https://github.com/user-attachments/assets/8f3300ee-5d1c-468f-b879-57f310f9b9ee">

Reply:
I think it's because there are escaping characters in those links if you copy pasted them from the PR.